compiler/cpp/src/generate/t_rs_generator.cc

```diff
@@ -450,7 +450,7 @@ string t_rs_generator::render_rs_type(t_type* type) {
     case t_base_type::TYPE_I64:
       return "i64";
     case t_base_type::TYPE_DOUBLE:
-      return "f64";
+      return "OrderedFloat<f64>";
```

Comment:
This will make interacting with floats somewhat of a bummer to wrap and unwrap them. Seems like it'd be better to use a use a map/set with a custom comparator like http://apasel422.github.io/tree/tree/map/struct.Map.html